### DISCUSSION TITLE
rendered view 模式下翻译，能否支持原文与译文共存？

### DISCUSSION THREAD

**Discussion by lingshaofu** [YiiGuxing/TranslationPlugin#3085] [Ideas]
![image](https://user-images.githubusercontent.com/17919814/216507659-375a4225-4137-4920-8065-4d64d3f40882.png)
如上图，现在 rendered view 模式下译文是直接替换原文的，但是译文经常词不达意，需要结合原文一起看才能理解含义。

现在我都是切换成 normal 模式再翻译的，如下图，这样一来不方便，二来 normal 模式下文本格式不易阅读。

![image](https://user-images.githubusercontent.com/17919814/216508273-3d1ea910-7d0f-40fb-972e-50a7b645ff9c.png)

所以非常希望能够支持 rendered view 模式下，译文与原文共存。

感谢大佬的付出！

**Comment by YiiGuxing** [comment]
可以到是可以，但是原文与译文直接一起显示的话显然并不是很好，这个需要再评估与调研。

**Reply by lingshaofu** [reply]
可以考虑加个开关控制~

**Reply by YiiGuxing** [reply]
需要已经在这里记录了，后面我会考虑的。

**Comment by zchazc** [comment]
### 前言
本来错开了一个issue来讨论这个，结果发现有个discussion，并且发现这个特性已经有人讨论了一部分了。所以在此处附上，对于作者顾虑的[“评估与调研”问题](https://github.com/YiiGuxing/TranslationPlugin/discussions/3085#discussioncomment-4858251)，从最近比较火的沉[浸式翻译(immersive-translate)](https://github.com/immersive-translate/immersive-translate)可以看出是有不少的需求的。

### 需求/建议描述

希望能够添加翻译后原文保留的设定。

### 建议的解决方案

在插件设置中添加“保留原文”开关，可以分功能，也可以直接是全局设定。
开启时，翻译结果包含原文。

### 附加信息

首先感谢:thumbsup:项目作者为广大程序员提供的便利工具，填补了很多的语言沟壑。
我之前读文档都是边查单词边读，遇到读不顺的就复制整段丢到翻译引擎中来回切换页面比对（双屏时就轻松些），毕竟翻译引擎再牛也无法完全传递原意。
最近比较火的沉[浸式翻译(immersive-translate)](https://github.com/immersive-translate/immersive-translate)中解决了这个痛点，觉得这个是个非常不错的特性，所以希望能够采纳。

**Reply by YiiGuxing** [reply]
文档所包含的参数表等部分目前还没有好的处理办法，这部分也不易区分，也暂时没有时间处理。
